modules/ml/src/main/java/org/apache/ignite/ml/composition/boosting/convergence/simple/ConvergenceCheckerStub.java

```diff
@@ -46,7 +46,6 @@
      * @param externalLbToInternalMapping External label to internal mapping.
      * @param loss Loss function.
      * @param datasetBuilder Dataset builder.
```

Comment:
I see `double precision` and `preprocessor` parameter which is not mentioned in javadoc.

Reply:
fixed